src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -86,6 +87,10 @@ enum RequisitionStatus {
         [CREATED, EDITING, APPROVED, PENDING_APPROVAL, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
+    static isApprovalStatus(RequisitionStatus status) {
+        return status in [APPROVED, REJECTED, PENDING_APPROVAL]
+    }
```

Comment:
I would make this more readable and call it something like `isInApprovalWorkflow` or something like that.
```
static isInApprovalWorkflow(RequisitionStatus status) { 
    return status in RequsitionStatus.listApproval()
}

static listApproval() { 
    [APPROVED, REJECTED, PENDING_APPROVAL]
}
```

Reply:
isInApprovalWorkflow doesn't sound right to me either, so if someone comes up with something better let me know.